.ci_support/build_all.py

```diff
@@ -98,6 +98,7 @@ def build_folders(recipes_dir, folders, arch, channel_urls):
 
     worker = {'platform': platform, 'arch': arch,
               'label': '{}-{}'.format(platform, arch)}
+    print(worker)
```

Comment:
This is unnecessary. You can already see this information,
```
Computed that there are 3 distributions to build from 1 recipes
Resolved dependencies, will be built in the following order:
     psutil-5.5.0-python_2.7-on-linux-64
     psutil-5.5.0-python_3.6-on-linux-64
     psutil-5.5.0-python_3.7-on-linux-64
```

Reply:
fixed